include-fragment-element.js

```diff
@@ -17,6 +17,10 @@ function handleData(el, data) {
       if (parentNode) {
         el.insertAdjacentHTML('afterend', html)
         parentNode.removeChild(el)
+
+        // Hack to get custom elements in html upgraded
+        // https://github.com/webcomponents/custom-elements/issues/104
+        parentNode.innerHTML = parentNode.innerHTML
```

Comment:
Would it be possible to avoid using `insertAdjacentHTML` in favor of `innerHTML` or another method of parsing/inserting HTML that is compatible with webcomponents?

Reply:
Here's an alternative: https://github.com/github/include-fragment-element/compare/muan/alt